x-pack/plugins/fleet/server/services/epm/packages/get_bulk_assets.ts

```diff
@@ -33,11 +33,9 @@ const getKibanaLinkForESAsset = (type: ElasticsearchAssetType, id: string): stri
     case 'data_stream_ilm_policy':
       return `/app/management/data/index_lifecycle_management/policies/edit/${id}`;
     case 'transform':
-      // TODO: Confirm link for transforms
-      return '';
+      return `/app/management/data/transform?_a=(transform:(queryText:${id}))`;
```

Comment:
Might not be possible, but adding the description of the transform, if set, would be nice here, as happens for dashboard assets:

<img width="465" alt="Screenshot 2024-08-02 at 16 15 59" src="https://github.com/user-attachments/assets/c1842a83-b8da-4e5c-b65b-87fb4e05314e">

Reply:
Currently for ES assets that isn't possible but I can create a follow up issue to allow it for ES assets.